integration-tests/api/__tests__/admin/price-list.js

```diff
@@ -795,8 +795,8 @@ describe("/admin/price-lists", () => {
         await simplePriceListFactory(dbConnection, {
           id: "test-list",
           prices: [
-            { variant_id: "test-variant-1", currency_code: "usd", amount: 100 },
-            { variant_id: "test-variant-4", currency_code: "usd", amount: 100 },
+            { variant_id: "test-variant-1", currency_code: "usd", amount: 150 },
```

Comment:
why does the amount changed here?

Reply:
because the default price that are set in `simplePriceListFactory`, it's just to be more heavyhanded wrt. what prices are from the price list in the tests